integration-tests/ledger-signer.spec.ts

```diff
@@ -111,9 +111,9 @@ CONFIGS().forEach(({ lib, setup, rpc }) => {
       });
     });
 
-    describe('Should sign operation with Ledger', () => {
+    describe('Test sign operation with Ledger', () => {
       jest.setTimeout(30000);
-      it('Should return the correct signature with the Ledger', async (done) => {
+      it('As a User I want to verify that Ledger returns the correct signature with the Ledger', async (done) => {
```

Comment:
Should we use the same format as the other files "Verify that..." without "As a User I want to"

Reply:
Changed to
'Verify that that Ledger will provide correct public key and public key hash for tz3 curve and path having 1 as account value'